py-polars/polars/show_versions.py

```diff
@@ -3,6 +3,7 @@
 import importlib
 import platform
 import sys
+import typing as _typing_
```

Comment:
Why not do `from typing import Callable, Any`? The underscores are not needed either?

Reply:
Good call. I will change it and import only specific elements of `typing`.